src/docs/patterns/overlay.mdx

```diff
@@ -1,46 +1,63 @@
-import { DemoCard, ImageGrid, MaterialDesignDescription } from '../../app/components';
-import BasicBottomsheet from '../../app/assets/design-patterns/action-list/action-list-overlay.png';
-import ComplexBottomsheet from '../../app/assets/design-patterns/bottom-sheet/complex-bottomsheet.png';
+import { DemoCard, ImageGrid, MaterialDesignDescription, Divider } from '../../app/components';
+import Banner from '../../app/assets/design-patterns/overlays/overlay-banner.svg';
+import DesktopDialog from '../../app/assets/design-patterns/overlays/desktop-dialog.png';
+import DesktopDialogTask from '../../app/assets/design-patterns/overlays/desktop-dialog-task.png';
 
-<!-- KEYWORDS: design pattern overlay modal sidebar popup dialog -->
+<!-- KEYWORDS: design pattern overlay modal sidebar popup dialog side sheet -->
 
-# Overlay Patterns
+# Overlays
 
-Overlays are used to present an immediate interaction to a user. These can be used for presenting information (such as with a Dialog) or capturing it (such as in a form). These elements come in a variety of shapes and sizes and can be either blocking (the user must interact with the overlay before they can do anything else) or non-blocking.
+<ImageGrid images={[Banner]} regularWidth />
 
-## Modal Dialog
+Overlays are useful in a variety of situations when you need to provide more context or information to a user than can be found in your primary content. For example, they can be used to present a confirmation dialog, details about a selected item, or actions that can be taken on a page. These can be used for presenting information (such as with a dialog) or capturing it (such as in a form).
+
+<Divider />
 
-The modal dialog is your typical pop-up window. Modals are great for presenting a single task to a user. By using a modal, you can allow the user to do more work without having to clutter your main layout with unnecessary information (e.g., an Add User button could open a modal with a form to enter the new user's information). These types of modals should be non-blocking, such that the user could click anywhere on the screen to dismiss it. Modals can also be used to edit large amounts of information that would be too much to fit into a sidebar.
+# Variations
 
-Modal dialogs are also very commonly used to confirm a user action (such as deleting an item). These usually appear in the center of the screen.
+Overlay elements come in a variety of shapes and sizes depending on their purpose and can be either blocking (i.e., the user must interact with the overlay before they can do anything else) or non-blocking.
+
+## Modal Dialog
 
-<MaterialDesignDescription
-    title={'Dialogs'}
-    url={'https://material.io/components/dialogs/'}
-    description={'Read more about dialogs on Material.io.'}
+<ImageGrid
+    images={[DesktopDialog, DesktopDialogTask]}
```

Comment:
For dialog boxes, stick with Material's Dialog pattern: https://material.io/components/dialogs#usage